pure-pso/crds/pso_v1alpha1_intrusion_crd.yaml

```diff
@@ -129,6 +129,95 @@ spec:
           type: object
   version: v1alpha1
   versions:
-  - name: v1alpha1
-    served: true
-    storage: true
+    - name: v1alpha1
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            apiVersion:
+              type: string
+            kind:
+              type: string
+            metadata:
+              type: object
+            spec:
+              properties:
+                replicas:
+                  items:
+                    properties:
+                      apiToken:
+                        type: string
+                      backendId:
+                        type: string
+                      backendType:
+                        enum:
+                          - Ephemeral
```

Comment:
While technically correct, do we want to document the `Ephemeral` and `PVC` sections publicly given we don't really support/test them at the moment?